stable/kommander-thanos/templates/federated-addon.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: types.kubefed.io/v1beta1
 kind: FederatedAddon
 metadata:
-  name: {{ template "kommander-thanos.fullname" . }}-proxy
```

Comment:
why don't you use the helpers.tpl to trunc the value there ?

Reply:
it's adding another `define` here:

https://github.com/mesosphere/charts/blob/dev/stable/kommander-thanos/templates/_helpers.tpl#L30-L32

Reply:
👍 I added a new variable for the truncated name.